Package.resolved

```diff
@@ -28,6 +28,15 @@
           "version": "3.3.2"
         }
       },
+      {
+        "package": "CryptoSwift",
```

Comment:
NOTE: I've included `CryptoSwift` to avoid depending on Vapor directly for the same reasons expressed in #7.